### ISSUE TITLE
DeprecationWarning

### ISSUE THREAD

**Issue by kjrajvir** [auttoio/typeform-node#3]
node:32571) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

When I put the code to check the form error I got this message.

unhandledRejection request to https://api.typeform.io/latest/forms/If7RA0 failed, reason: getaddrinfo ENOTFOUND api.typeform.io api.typeform.io:443

**Comment by yanneves** [comment]
Indeed, the typeform.io API this library was built for no longer exists - it has since moved and documentation can be found at https://developer.typeform.com/

I no longer maintain this library and don't have plans to add support for the new Typeform API

**Comment by picsoung** [comment]
Hi 👋 
I am Nicolas from Typeform, where I work as a Developer Advocate.
Thank you @yanneves for all your work maintaining this library.
We indeed deprecated our old API to give you something new and shiny with more features.

If you are a node Developer we a new Node library for this API https://www.npmjs.com/package/@typeform/api-client#

Hope you like it :)

**Comment by yanneves** [comment]
Thanks Nicolas! Glad to hear you've built an official client. I'll update this package with a 'no longer maintained' warning and link to your official client.
